gradle-jdks/src/main/java/com/palantir/gradle/jdks/JdksPlugin.java

```diff
@@ -42,76 +37,21 @@ public void apply(Project rootProject) {
         }
         rootProject.getPluginManager().apply(BaselineJavaVersions.class);
```

Comment:
I think we only need to apply `BaselineJavaVersions` if it's the `BaselineJavaJdksPlugin` now? Could even just apply it there.